client/views/msg.tpl

```diff
@@ -1,6 +1,6 @@
 <div class="msg {{type}}{{#if self}} self{{/if}}{{#if highlight}} highlight{{/if}}" id="msg-{{id}}" data-time="{{time}}" data-from="{{from}}">
-	<span class="time tooltipped tooltipped-e" aria-label="{{localetime time}}">
-		{{tz time}}
+	<span class="time tooltipped tooltipped-e {{#if seconds}}seconds{{/if}}" aria-label="{{localetime time}}">
```

Comment:
If we cared about it updating correctly when we change the settings (currently if you change the settings only new messages will have the correct format) then we could set it on the body (or container) and then have both time formats rendered, and show/hide in css based on the parameter we set on the container. That doesn't reaaaaally work well if we end up having multiple settings, and maybe feels a bit overkill (it doubles the number of time spans we have, with half of them being hidden). Let's leave that for now, this is good enough for just now I think. This is a setting we should have had a long time ago, personally.